src/06-loops.js

```diff
@@ -11,7 +11,13 @@
  *  //> 25
  *  (3 + 4 + 5 + 6 + 7 = 25)
  */
-function rangeSum(min, max) {}
+ function rangeSum(min, max) {
+  let total = 0 // we start here at Zero
+     for (let i = min; i <= max; i++){ //This loop starts the process at min and ends at Max
+         total = total + i // here we take the total in line 15 and which starts at Zero and we add the current value of i to the value of total until we reach the stopping condition
```

Comment:
I love the comments.  The best convention for comments is to put the comments above the code.  It's easier to read.